snap/info_snap_yaml.go

```diff
@@ -324,6 +335,14 @@ func setAppsFromSnapYaml(y snapYaml, snap *Info) error {
 			app.Slots[slotName] = slot
 			slot.Apps[appName] = app
 		}
+		for name, data := range yApp.Sockets {
+			app.Sockets[name] = &SocketInfo{
+				App:          app,
+				Name:         name,
+				ListenStream: data.ListenStream,
```

Comment:
Note that systemd actually supports [multiple listen streams](https://www.freedesktop.org/software/systemd/man/systemd.socket.html), to quote:

> These options may be specified more than once, in which case incoming traffic on any of the sockets will trigger service activation, and all listed sockets will be passed to the service, regardless of whether there is incoming traffic on them or not. If the empty string is assigned to any of these options, the list of addresses to listen on is reset, all prior uses of any of these options will have no effect.

Should we actually be supporting a list of listen-streams, then?

Reply:
This was initially suggested by @niemeyer too, but it was agreed to have a `.socket` file for each socket the application wants to listen on.

Reply:
If I want to declare an app that is activated by multiple sockets, what does that YAML look like, then?

Reply:
Oh, multiple socket names I suppose.

Reply:
Correct, you can use multiple entries